book/book.tex

```diff
@@ -29,6 +29,11 @@
 \usepackage{kantlipsum}
 \usepackage{cleveref}
 
+\usepackage[version=4]{mhchem}
```

Comment:
Is this package strictly necessary? I see that it is used for e. g
```
\ce{H+}
```
but would it be possible to instead use e.g
```
$\mathrm{H}^{+}$
```
?

Reply:
It can be done that way.  mchem is a neat package for chemical formulae, but easy to format manually in this case